src/app/Http/Controllers/Auth/LoginController.php

```diff
@@ -37,4 +37,8 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    public function index() {
```

Comment:
This is an API, so you don't have the need for an index page.

Reply:
Fixed in 82175c3